### DISCUSSION TITLE
Convert articles made with Guthenberg to Elementor

### DISCUSSION THREAD

**Discussion by DotCat1985** [elementor/elementor#18581] [Developers Community]
Hi,
I activated **Elementor** plugin on my **WordPress** website containing hundreds of articles made with **Guthenberg**.
How can I automatically convert those articles from **Guthenberg** to **Elementor**?
I wouldn't like to waste much time to convert them manually.

**Comment by WebsiteDons** [comment]
There is no simple single method to bulk convert posts/pages within elementor's tool box. However there may be a third-party plugin in existence since it's likely other people have the same desire.

The conversion process is quite complex since there are multiple rows created in each post's `postmeta` database table. The basic rows to make a page editable with elementor are

```
meta_key _elementor_edit_mode
meta_value builder

meta_key _elementor_template_type
meta_value wp-page

meta_key _elementor_version
meta_value 3.5.6 //set to current version installed
```

You could then create a process to loop through each post and update the `postmeta` table with those rows. To get an idea of the basic rows created, manually set a single post to **edit with elementor** then view that post's `postmeta` columns for `_elementor` values to see what was inserted.
